pokemon-showdown

```diff
@@ -86,8 +86,8 @@ if (!process.argv[2] || /^[0-9]+$/.test(process.argv[2])) {
 	case 'start':
 		{
 			process.argv.splice(2, 1);
-			if (process.argv[2] === '--skip-build') {
-				process.argv.splice(2, 1);
+			if (process.argv.includes('--skip-build')) {
+				process.argv.splice(process.argv.indexOf('--skip-build'), 1);
```

Comment:
You don't have the splice for `--no-security`. I suspect it might not be necessary.

Reply:
Probably should have all the argv's in one place?